src/world.c

```diff
@@ -1,6 +1,16 @@
 #include "config.h"
 #include "noise.h"
 #include "world.h"
+#include "stdlib.h"
+
+int chooseTree(int choice){
```

Comment:
Perhaps a better name for this function would be chooseTreeSize, since when it is used on line 59 of this file, treeSize is begin set to the result of this function